selfdrive/controls/controlsd.py

```diff
@@ -192,6 +194,11 @@ def update_events(self, CS):
       self.events.add(EventName.controlsInitializing)
       return
 
+    # Disable on rising edge of gas or brake. Also disable on brake when speed > 0
+    if (CS.gasPressed and self.active) or (CS.brakePressed and (not self.brake_pressed_prev or not CS.standstill)):
+      self.events.add(EventName.pedalPressed)
```

Comment:
let's save the previous car state so we can do the same check. it's unclear that `CS.gasPressed and self.active` enforces `Disable on rising edge of gas`